tests/fixtures/encoder_decoder/wikitables_semantic_parser_with_mixture_feedforward/serialization/vocabulary/non_padded_namespaces.txt

```diff
@@ -0,0 +1,2 @@
+*tags
```

Comment:
You shouldn't need to add a `serialization` directory here - you only need that to test `Predictors`.  I think these fixtures are just large unused files.

Reply:
done.